ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/coordinator/BasicAction.java

```diff
@@ -3386,6 +3386,19 @@ private final boolean checkChildren (boolean isCommit)
                     tsLogger.i18NLogger.warn_coordinator_BasicAction_57(get_uid());
                 } else {
                     tsLogger.i18NLogger.warn_coordinator_BasicAction_58(get_uid());
+
```

Comment:
Its a good observation but this is warn, rather than trace. In terms of string contatenation I am not really sure how I can avoid it here?